src/components/DownloadMailingListButton.tsx

```diff
@@ -17,31 +16,40 @@ export function DownloadMailingListButton (props: DownloadButtonProps) {
         let email
         const propertyAddress = propertyObject.Deal_Name
         const ownerDetails = propertyObject.owner_details
-        const contact: OwnerType = ownerDetails[0]
-        let owner: OwnerType = ownerDetails[1]
+        const relatedContact: OwnerType = ownerDetails[0]
```

Comment:
how can you know that it is always index 0? There may be more than 2 owners

Reply:
@haydenf?

Reply:
I didn't think of this, do properties have multiple owners? If so I'm unsure of how to go about a fix for it

Reply:
I'd suggest using the code I had in other places where it uses ownerDetails.find to locate

Reply:
Still an issue